samples/3d-platformer/build.gradle.kts

```diff
@@ -0,0 +1,14 @@
+plugins {
+    kotlin("multiplatform") version "1.3.71"
```

Comment:
why not 72?

Reply:
same as mini-mages, but I'll try updating.